xbmc/guilib/GUIBaseContainer.cpp

```diff
@@ -557,12 +559,12 @@ void CGUIBaseContainer::OnPrevLetter()
   }
 }
 
-void CGUIBaseContainer::OnJumpLetter(char letter, bool skip /*=false*/)
+void CGUIBaseContainer::OnJumpLetter(std::string letter, bool skip /*=false*/)
 {
   if (m_matchTimer.GetElapsedMilliseconds() < letter_match_timeout)
-    m_match.push_back(letter);
+    m_match += letter;
   else
-    m_match = StringUtils::Format("%c", letter);
+    m_match = letter;
```

Comment:
```suggestion
    m_match = std::move(letter);
```

Reply:
I'm not sure I want to `std::move` the letter into m_match.
All the way down in the code, m_match is cleared but letter is still used unchanged:
```
  // no match found - repeat with a single letter
  std::wstring wmatch;
  g_charsetConverter.utf8ToW(m_match, wmatch);
  if (wmatch.length() > 1)
  {
    m_match.clear();
    OnJumpLetter(letter, true);
  }
```

Reply:
I missed that, nevermind